### ISSUE TITLE
Refactor preview: elements should have better aria label

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#88554]
Use a screen reader to navigate over the refactor preview elements 
1. It reads what is actually written
2. It should be more user friendly and read what is gettign replaced by what where. The Search Find / Replace experience nicely reads out concise understandable sentences. So you can use that as a good example
